test/mpi/rma/win_shared_zerobyte.c

```diff
@@ -46,6 +46,13 @@ int main(int argc, char **argv)
         ELEM_PER_PROC = 16;
     }
 
+    /* Obtain the world rank of last process in my shm_comm. */
+    if (shm_rank == shm_nproc - 1)
+        last_shm_rank_w = rank;
```

Comment:
A world rank gives an unique init value for each shm_comm. This is useful when multiple `shm_comm` exists on a node (e.g., `ODD_EVEN`) and the shared buffers incorrectly overlap (such as the bug fixed in the next patch). The world rank based way can hint whether overlap happens, but `shm_rank` based way cannot.